standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/Msck.java

```diff
@@ -108,15 +108,16 @@ public int repair(MsckInfo msckInfo) {
     boolean success = false;
     long txnId = -1;
     int ret = 0;
+    long partitionExpirySeconds = msckInfo.getPartitionExpirySeconds();
```

Comment:
this doesn't seem right; see other comment as well

Reply:
fixed